readme.md

```diff
@@ -0,0 +1,10 @@
+ros package can be built with catkin.
+
+Dependencies:
+- librealsense
+- pyrealsense
+- mxnet
+- opencv
+
+run face detection:
+python face_detection.py
```

Comment:
it would be nice to have an overview of the features included in the packages. some small description wouldn't harm